### ISSUE TITLE
Pytrie fails on python 3.8

### ISSUE THREAD

**Issue by scarabeusiv** [gsakkis/pytrie#5]
Direct collection imports are no longer working on python 3.8 and newer and need to be tweaked to keep working:

```
[  128s]   /usr/lib/python3.7/site-packages/pytrie.py:48: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
[  128s]     from collections import MutableMapping

```

**Comment by shevron** [comment]
Sorry to nag, but is there a (pypi) release planned with this fix, now that 3.9 is out and users will soon start upgrading?

**Comment by gsakkis** [comment]
Just pushed a new release on pypi.
